lib/commands/general.js

```diff
@@ -116,6 +116,12 @@ commands.keys = async function (keys) {
     }
     await this.setValue(keys, el.ELEMENT);
   } else {
+    if (_.isArray(keys)) {
+      keys = keys.join('');
+    }
+    if (!_.isString(keys)) {
+      keys = `${keys}`;
```

Comment:
It is, as long as `keys` is an array with a string inside. I think the spec says it ought to take an array of characters, in which case `['a', 'b', 'c'].toString() == 'a,b,c'`